pylint_django/tests/input/func_noerror_foreign_key_in_non_django_class.py

```diff
@@ -0,0 +1,25 @@
+"""
+Checks that Pylint raise error when a 'ForeignKey' appears in a
+non-django class
+
+The real case is described as follow:
+The project use tastypie and django.
+tastypie has a `ForeignKey` field which has the same name
+as django's `ForeignKey`.
+The issue is the lint trys resolving the `ForeignKey` for the
+tastypie `ForeignKey` which cause import error.
+"""
+# pylint: disable=missing-docstring
+
+class ForeignKey: # pylint: disable=too-few-public-methods
+    def test_methond(self):
+        pass
+
+
+class Resource: # pylint: disable=too-few-public-methods
+    def test_methond(self):
+        pass
```

Comment:
I think you can add tastypie as a testing dependency. See `extras_require` in `setup.py` and `pylint_django/tests/input/external_factory_boy_noerror.py` for example.

Reply:
Thanks, updated!